main.js

```diff
@@ -165,9 +166,12 @@ options = {
 	port: config.get("minecraftserver.port"),
 	version: config.get("minecraftserver.version")
 }
-if (config.get("antiAntiAFK")) setInterval(function () {
-	if(proxyClient == null && webserver.isInQueue && finishedQueue) client.write("chat", { message: "!que", position: 1 })
-}, 50000)
+
+function startAntiAntiAFK(){
+	if (!config.get("antiAntiAFK").get("enabled")) return;
```

Comment:
change this line to 
```if (!config.has("antiAntiAFK.enabled") || !config.get("antiAntiAFK.enabled")) return;```
because the other way it will error when antiAntiAFK is on the old config.

Reply:
Okey-dokey